examples/java-matter-controller/java/src/com/matter/controller/Main.java

```diff
@@ -17,8 +17,11 @@
  */
 package com.matter.controller;
 
+import chip.devicecontroller.ChipDeviceController;
+
 public class Main {
   public static void main(String[] args) {
+    ChipDeviceController controller = new ChipDeviceController(null);
```

Comment:
I think null will cause a native crash. CHIPDeviceController-JNI newDeviceController() doesn't add any special handling for null. Can you use ControllerParams? 

```
ChipDeviceController controller = new ChipDeviceController(
  ControllerParams.newBuilder()
    .setUdpListenPort(0)
    .setControllerVendorId(0xFFF1)
    .build());
```

Reply:
plan to have ControllerParams, currently it is just to put one line code here and check whether it can be compiled and run without crash. thanks